torch/_inductor/kernel/flex_attention.py

```diff
@@ -790,10 +789,10 @@ def flex_attention_backward(*args, **kwargs):
             call_sizes=query.get_size() + [key.get_size()[2]],
             num_stages=num_stages,
             num_warps=num_warps,
-            BLOCK_M1=BLOCK_M,
-            BLOCK_N1=BLOCK_N,
-            BLOCK_M2=BLOCK_N,
-            BLOCK_N2=BLOCK_M,
+            BLOCK_M1=BLOCK1,
+            BLOCK_N1=BLOCK1,
+            BLOCK_M2=BLOCK2,
+            BLOCK_N2=BLOCK2,
```

Comment:
The constraints in the kernel: ```BLOCK_N1``` must be a multiple of ```BLOCK_M1```, and ```BLOCK_M2``` must be a multiple of ```BLOCK_N2```. I benchmarked a bit and found the best performance one is ```multiplier == 1``` for both, so I just make them equal. I'm thinking probably in the future we should add two ```multipliers``` in the autotune config if it's needed.